client/src/components/DocumentPage/VersionHistoryModal.jsx

```diff
@@ -58,20 +58,21 @@ export default function VersionHistoryModal({ documentID, onClose, quillRef }) {
       console.log("Revert response", response);
 
       if (response && response.updatedContent) {
-        // Clear editor before applying revert patch
-        quillRef.current.setContents([response.updatedContent]);
```

Comment:
Based on this comment, was this a bug? Because it says "clear editor" but it was actually setting the content?